modules/logging-release-notes-5-8-0.adoc

```diff
@@ -0,0 +1,64 @@
+//module included in logging-5-8-release-notes.adoc
+:content-type: REFERENCE
+[id="logging-release-notes-5-8-0_{context}"]
+= Logging 5.8.0
+This release includes link:https://access.redhat.com/errata/RHBA-2023:6139[OpenShift Logging Bug Fix Release 5.8.0] and link:https://access.redhat.com/errata/RHBA-2023:6134[OpenShift Logging Bug Fix Release 5.8.0 Kibana].
+
+[id="logging-release-notes-5-8-0-deprecation-notice"]
+== Deprecation notice
+In Logging 5.8, Elasticsearch, Fluentd, and Kibana are deprecated and are planned to be removed in Logging 6.0, which is expected to be shipped alongside {product-title} version 4.16. Red Hat will provide critical and above CVE bug fixes and support for these components during the current release lifecycle, but these components will no longer receive feature enhancements. The Vector-based collector provided by the Cluster Logging Operator, and LokiStack provided by the Loki Operator are the preferred operators for log collection and storage.  We encourage all users to adopt the Vector and Loki log stack, as this will be the stack that will be enhanced going forward.
+
+[id="logging-release-notes-5-8-0-enhancements"]
+== Enhancements
+
+[id="logging-release-notes-5-8-0-log-collection"]
+=== Log Collection
+* With this update, you can deploy multiple, isolated, and RBAC-protected `ClusterLogForwarder` custom resource (CR) instances in any namespace. This allows independent groups to forward desired logs to any destination while isolating their configuration from other collector deployments. (link:https://issues.redhat.com/browse/LOG-1343[LOG-1343])
+
+[IMPORTANT]
+====
+In order to support multi-cluster log forwarding in additional namespaces other than the `openshift-logging` namespace, you must update the Cluster Logging Operator to watch all namespaces. This functionality is supported by default in new Cluster Logging Operator version 5.8 installations.
+====
+
+* With this update, you can use the flow control or rate limiting mechanism to limit the volume of log data that can be collected or forwarded by dropping excess log records. The input limits prevent badly-behaved containers from overloading the {logging-uc} and the output limits put a ceiling on the rate of logs shipped to a given data store. (link:https://issues.redhat.com/browse/LOG-884[LOG-884])
```

Comment:
I'm assuming that "badly-behaved containers" is the technical term for these naughty instances of software? 😂 

In any case, if you can replace "badly-behaved" with some other phrasing that doesn't [anthropomorphize](https://www.ibm.com/docs/en/ibm-style?topic=grammar-anthropomorphism) the container as much, I'd suggest using that instead.

Reply:
@skopacz1  How about "poorly-performing" containers? Please feel free to suggest any other phrase that suits better. :)